### PR TITLE
Feature/merge release 2.1 to master

### PR DESCRIPTION
**Pull request by mikonieminen**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [0.2.1] - 2020-12-07
+### Fixed
+- Upgrade to `colombe` version `0.4.0`, this fixes vanishing full stop issue,
```
Comment (aronerben) [inline CHANGES.md:5]: Shouldn't this be `0.5.0`?

**Review by aronerben — CHANGES_REQUESTED** [review]
Please also run `opam lock` to update the lockfile (should fix pipeline)

File: CHANGES.md (hunk 1 above)
Comment (mikonieminen) [reply CHANGES.md:5]: No, this is old release commit we merged back to master. Next release should probably be `0.3.0` or `0.2.2`.

**Comment by mikonieminen** [comment]
> Please also run `opam lock` to update the lockfile (should fix pipeline)

I'll update that one once we have a working version in `master`.

File: CHANGES.md (hunk 1 above)
Comment (aronerben) [reply CHANGES.md:5]: Ok, we should release those soon, so there is no desync between the versions in the changelog and the versions we use.

**Review by aronerben — APPROVED** [review]

**Pull request merged**